test/integration/master/synthetic_master_test.go

```diff
@@ -116,6 +119,15 @@ func TestKubernetesService(t *testing.T) {
 }
 
 func TestEmptyList(t *testing.T) {
+	testEmptyList(t)
+}
+
+func TestEmptyListWithoutSelfLink(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.RemoveSelfLink, true)()
```

Comment:
if the default is already true... shouldn't this test have already failed?

Reply:
oh, this is 1.19

do we normally backport bugfixes for alpha features like this?

Reply:
I think it depends - in this case it's about backward compatibility, so I think it's worth backporting. WDYT?

Reply:
I would tend to avoid backports for alpha features unless they were fixing something like data loss or a severe security issue

Reply:
OK - I don't have strong opinion here.
I will keep that open for a couple more days if someone else have other arguments and close otherwise.

Reply:
@liggitt how about upgrade scenario? Suppose I have cluster with 1.19 deployed, and would like to switch to 1.20, but with knowledge, that one of the components need to be corrected to not use `selfLink`, I have no option for test it properly against that particular change. Especially, that obviously it's a bug, which makes that feature gate useless/faulty.